src/irvisualizer/html_template_StmtToViz_dependencies.html

```diff
@@ -0,0 +1,4 @@
+<!-- Hierarchy links -->
+<link rel='stylesheet' href='https://unpkg.com/treeflex/dist/css/treeflex.css'>
+
+<script src='http://code.jquery.com/jquery-1.10.2.js'></script>
```

Comment:
Please upgrade to JQuery 3.7.x in order to resolve CVEs. I can help flush out deprecated JQuery API calls after that.

Reply:
I will nuke StmtToViz. I started by separating the two first cleanly to work on the StmtHTML. Thanks for the reminder tho!